consensus/e2e_tests/state_sync_test.go

```diff
@@ -0,0 +1,161 @@
+package e2e_tests
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/benbjohnson/clock"
+	typesCons "github.com/pokt-network/pocket/consensus/types"
+	"github.com/pokt-network/pocket/shared/codec"
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/anypb"
+)
```

Comment:
This might be too much, but have you lookws into how hard it would be to create a small unit test just for the StateSyncServer module, without creating a whole consensus mock?

I feel like you might be able to reuse some of the mocks, but lmk if you think its too much work.

Reply:
I will create a ticket for this.